main.py

```diff
@@ -178,11 +178,9 @@ def fetch_checks_for_package(package_name):
 
 def get_project_urls(info: dict) -> list[tuple[str, str, str]]:
     names_urls = [
-        ("bugtrack_url", info.get("bugtrack_url")),
         ("docs_url", info.get("docs_url")),
         ("Downloads", info.get("download_url")),
```

Comment:
Should we rename these to match `docs_url` to be `download_url` and `home_page`?

Reply:
Sorry for the late reply, was at the Python core dev sprint last week in Mountain View :)

I think `Homepage` is more useful than `home_page`, as `Homepage` is what PyPI/Warehouse offers to the end user.

For example, Pillow sets `url`:

```cfg
url = https://python-pillow.org/
```

https://github.com/python-pillow/Pillow/blob/e0c30d2fcd90ed9cfb1bdf2d27f6170bc9e762ad/setup.cfg#L6

And six others in `project_urls`:

```cfg
project_urls =
    Documentation=https://pillow.readthedocs.io/
    Source=https://github.com/python-pillow/Pillow
    Funding=https://tidelift.com/subscription/pkg/pypi-pillow?utm_source=pypi-pillow&utm_medium=pypi
    Release notes=https://pillow.readthedocs.io/en/stable/releasenotes/index.html
    Changelog=https://github.com/python-pillow/Pillow/blob/main/CHANGES.rst
    Twitter=https://twitter.com/PythonPillow
```

https://github.com/python-pillow/Pillow/blob/e0c30d2fcd90ed9cfb1bdf2d27f6170bc9e762ad/setup.cfg#L28-L34

And `url` is converted into "Homepage" by PyPI for display at https://pypi.org/project/Pillow/ with the other six:

<img width="219" alt="image" src="https://user-images.githubusercontent.com/1324225/195053077-9022fb2d-bbb3-4d57-906b-37366373c6df.png">

And similarly for API at https://pypi.org/pypi/Pillow/json :

```json
"project_urls": {
"Changelog": "https://github.com/python-pillow/Pillow/blob/main/CHANGES.rst",
"Documentation": "https://pillow.readthedocs.io/",
"Funding": "https://tidelift.com/subscription/pkg/pypi-pillow?utm_source=pypi-pillow&utm_medium=pypi",
"Homepage": "https://python-pillow.org/",
"Release notes": "https://pillow.readthedocs.io/en/stable/releasenotes/index.html",
"Source": "https://github.com/python-pillow/Pillow",
"Twitter": "https://twitter.com/PythonPillow"
},
```

As a pypi-data user, I'm interested in searching what is available from PyPI (API or display).

Reply:
And similarly for `download_url` -> `Downloads` (mentioned in https://github.com/sethmlarson/pypi-data/pull/16; Pillow doesn't use that one, but I can look up some examples if you like).

Reply:
@hugovk That all sounds good to me, could you create a PR for this?

Reply:
Yup, please see PR https://github.com/sethmlarson/pypi-data/pull/20